ftplugin/markdown.vim

```diff
@@ -105,8 +105,17 @@ endfunction
 function! s:GetHeadingLinkGFM(headingName)
     let l:headingLink = tolower(a:headingName)
 
+    " \%^ : beginning of file
```

Comment:
Sorry but I have no experience dealing with this situation, and I checked help documents, did not found something helpful.